### ISSUE TITLE
Could you please specify the dependencies of this project and what versions they are?

### ISSUE THREAD

**Issue by 57potato** [Cocofeat/TBraTS#12]

**Comment by Cocofeat** [comment]
Thanks for your interest,
Here are the ref:
channels:
  - pytorch
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - asttokens=3.0.0=pyhd8ed1ab_0
  - backcall=0.2.0=pyh9f0ad1d_0
  - blas=1.0=mkl
  - brotli-python=1.0.9=py38h6a678d5_8
  - bzip2=1.0.8=h5eee18b_6
  - ca-certificates=2024.12.14=hbcca054_0
  - certifi=2024.8.30=pyhd8ed1ab_0
  - charset-normalizer=3.3.2=pyhd3eb1b0_0
  - comm=0.2.2=pyhd8ed1ab_0
  - cudatoolkit=11.3.1=h2bc3f7f_2
  - debugpy=1.6.7=py38h6a678d5_0
  - decorator=5.1.1=pyhd8ed1ab_0
  - entrypoints=0.4=pyhd8ed1ab_0
  - executing=2.1.0=pyhd8ed1ab_0
  - ffmpeg=4.3=hf484d3e_0
  - freetype=2.12.1=h4a9f257_0
  - gmp=6.2.1=h295c915_3
  - gnutls=3.6.15=he1e5248_0
  - idna=3.7=py38h06a4308_0
  - intel-openmp=2023.1.0=hdb19cb5_46306
  - ipykernel=6.29.5=pyh3099207_0
  - ipython=8.12.2=pyh41d4057_0
  - jedi=0.19.1=pyhd8ed1ab_0
  - jpeg=9e=h5eee18b_3
  - jupyter_client=7.3.4=pyhd8ed1ab_0
  - jupyter_core=5.7.2=pyh31011fe_1
  - lame=3.100=h7b6447c_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.40=h12ee557_0
  - lerc=3.0=h295c915_0
  - libdeflate=1.17=h5eee18b_1
  - libffi=3.4.4=h6a678d5_1
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libiconv=1.16=h5eee18b_3
  - libidn2=2.3.4=h5eee18b_0
  - libpng=1.6.39=h5eee18b_0
  - libsodium=1.0.18=h36c2ea0_1
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtasn1=4.19.0=h5eee18b_0
  - libtiff=4.5.1=h6a678d5_0
  - libunistring=0.9.10=h27cfd23_0
  - libuv=1.48.0=h5eee18b_0
  - libwebp-base=1.3.2=h5eee18b_1
  - lz4-c=1.9.4=h6a678d5_1
  - matplotlib-inline=0.1.7=pyhd8ed1ab_0
  - mkl=2023.1.0=h213fc3f_46344
  - mkl-service=2.4.0=py38h5eee18b_1
  - mkl_fft=1.3.8=py38h5eee18b_0
  - mkl_random=1.2.4=py38hdb19cb5_0
  - ncurses=6.4=h6a678d5_0
  - nest-asyncio=1.6.0=pyhd8ed1ab_0
  - nettle=3.7.3=hbbd107a_1
  - numpy-base=1.24.3=py38h060ed82_1
  - openh264=2.1.1=h4ff587b_0
  - openjpeg=2.5.2=he7f1fd0_0
  - openssl=3.0.15=h5eee18b_0
  - parso=0.8.4=pyhd8ed1ab_0
  - pexpect=4.9.0=pyhd8ed1ab_0
  - pickleshare=0.7.5=py_1003
  - pip=24.2=py38h06a4308_0
  - platformdirs=4.3.6=pyhd8ed1ab_0
  - prompt-toolkit=3.0.48=pyha770c72_0
  - prompt_toolkit=3.0.48=hd8ed1ab_1
  - ptyprocess=0.7.0=pyhd3deb0d_0
  - pure_eval=0.2.3=pyhd8ed1ab_0
  - pygments=2.18.0=pyhd8ed1ab_0
  - pysocks=1.7.1=py38h06a4308_0
  - python=3.8.20=he870216_0
  - python_abi=3.8=2_cp38
  - pytorch=1.11.0=py3.8_cuda11.3_cudnn8.2.0_0
  - pytorch-mutex=1.0=cuda
  - pyzmq=25.1.2=py38h6a678d5_0
  - readline=8.2=h5eee18b_0
  - requests=2.32.3=py38h06a4308_0
  - setuptools=75.1.0=py38h06a4308_0
  - six=1.16.0=pyh6c4a22f_0
  - sqlite=3.45.3=h5eee18b_0
  - stack_data=0.6.2=pyhd8ed1ab_0
  - tbb=2021.8.0=hdb19cb5_0
  - tk=8.6.14=h39e8969_0
  - torchaudio=0.11.0=py38_cu113
  - torchvision=0.12.0=py38_cu113
  - tornado=6.1=py38h0a891b7_3
  - traitlets=5.14.3=pyhd8ed1ab_0
  - typing_extensions=4.11.0=py38h06a4308_0
  - urllib3=2.2.3=py38h06a4308_0
  - wcwidth=0.2.13=pyhd8ed1ab_0
  - wheel=0.44.0=py38h06a4308_0
  - xz=5.4.6=h5eee18b_1
  - zeromq=4.3.5=h6a678d5_0
  - zlib=1.2.13=h5eee18b_1
  - zstd=1.5.6=hc292b87_0
  - pip:
      - absl-py==2.1.0
      - appdirs==1.4.4
      - argparse==1.4.0
      - art==6.3
      - batchgenerators==0.25.1
      - cachetools==5.5.0
      - click==8.1.7
      - coloredlogs==15.0.1
      - contextlib2==21.6.0
      - contourpy==1.1.1
      - cycler==0.12.1
      - dictances==1.5.3
      - docker-pycreds==0.4.0
      - einops==0.6.1
      - filelock==3.16.1
      - flatbuffers==24.12.23
      - fonttools==4.54.1
      - fsspec==2024.10.0
      - future==1.0.0
      - gitdb==4.0.11
      - gitpython==3.1.43
      - google-auth==2.35.0
      - google-auth-oauthlib==1.0.0
      - grpcio==1.67.0
      - h5py==3.11.0
      - hausdorff==0.2.6
      - huggingface-hub==0.26.2
      - humanfriendly==10.0
      - imageio==2.35.1
      - importlib-metadata==8.5.0
      - importlib-resources==6.4.5
      - joblib==1.4.2
      - kiwisolver==1.4.7
      - lazy-loader==0.4
      - linecache2==1.0.0
      - llvmlite==0.38.1
      - markdown==3.7
      - markupsafe==2.1.5
      - matplotlib==3.7.2
      - medpy==0.5.2
      - miseval==1.2.2
      - ml-collections==0.1.1
      - monai==1.2.0
      - mpmath==1.3.0
      - munkres==1.1.4
      - networkx==3.1
      - nibabel==5.2.1
      - numba==0.55.2
      - numpy==1.24.4
      - oauthlib==3.2.2
      - onnxruntime==1.19.2
      - opencv-contrib-python==4.8.0.76
      - opencv-python==4.7.0.72
      - packaging==24.1
      - pandas==2.0.3
      - pathtools==0.1.2
      - pillow==9.5.0
      - protobuf==4.25.5
      - psutil==6.1.0
      - ptflops==0.7.3
      - pyasn1==0.6.1
      - pyasn1-modules==0.4.1
      - pycm==4.1
      - pymia==0.3.4
      - pyparsing==3.0.9
      - python-dateutil==2.9.0.post0
      - pytz==2024.2
      - pywavelets==1.4.1
      - pyyaml==6.0
      - requests-oauthlib==2.0.0
      - rsa==4.9
      - safetensors==0.4.5
      - scikit-image==0.21.0
      - scikit-learn==1.2.2
      - scipy==1.10.1
      - seaborn==0.12.2
      - sentry-sdk==2.17.0
      - setproctitle==1.3.3
      - simpleitk==2.4.0
      - smmap==5.0.1
      - surface-distance-based-measures==0.1
      - sympy==1.13.3
      - tensorboard==2.14.0
      - tensorboard-data-server==0.7.2
      - tensorboard-plugin-wit==1.8.1
      - tensorboardx==2.6.2.2
      - thop==0.1.1-2209072238
      - threadpoolctl==3.5.0
      - tifffile==2023.7.10
      - timm==1.0.11
      - torchprofile==0.0.4
      - tqdm==4.65.0
      - traceback2==1.4.0
      - tzdata==2024.2
      - unittest2==1.1.0
      - wandb==0.15.4
      - werkzeug==3.0.6
      - zipp==3.20.2
